samples/bluetooth/broadcast_audio_source/boards/nrf5340_audio_dk_nrf5340_cpuapp.conf

```diff
@@ -1,4 +1,8 @@
-# The LC3 codec uses a large amount of stack. This app runs the codec in the work-queue, hence
-# inctease stack size for that thread.
-CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=4096
 CONFIG_MAIN_STACK_SIZE=4096
+
+# Use USB Audio as input
+CONFIG_USE_USB_AUDIO_INPUT=y
+CONFIG_USB_DEVICE_PRODUCT="Zephyr Broadcast Source"
```

Comment:
Would be interesting to find a way to set `CONFIG_USB_DEVICE_PRODUCT="Zephyr Broadcast Source"` for any device that enables USB, and not just the nRF5340 ADK